css/lookbook.scss

```diff
@@ -53,6 +53,39 @@ h1 {
 	object-fit: cover;
 }
 
+
+.lightbox-hidden {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	position: fixed;
+	background-color: rgba(0, 0, 0, 0.8);
+	/*background-color: $offblack;*/
```

Comment:
Nah. I did that at first, but then it makes the entire div 80% opacity, not just the bg.